app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivity.kt

```diff
@@ -16,14 +16,16 @@ const val SELECTED_CONTROLS_TITLE_SAVED_KEY =
 const val LAST_LOADED_FRAGMENT_KEY = "LAST_LOADED_FRAGMENT_KEY"
 const val PROFILE_LIST_FRAGMENT = "PROFILE_LIST_FRAGMENT"
 const val APP_VERSION_FRAGMENT = "APP_VERSION_FRAGMENT"
+const val LOGOUT_DIALOG_FRAGMENT = "LOGOUT_DIALOG_FRAGMENT"
```

Comment:
This tag should be shifted to `LogoutDialogFragment` companion object and then it should be used wherever requried.

Reply:
Done